Sources/EpoxyCollectionView/Models/ItemModel/AnyItemModel.swift

```diff
@@ -33,6 +33,10 @@ public struct AnyItemModel: EpoxyModeled {
 
 }
 
+// MARK: ErasedContentProviding
```

Comment:
@thedrick I see that you added this file. Do you have any thoughts on this change?

Reply:
@bachand no comments, it's entirely possible that this file used to be something totally different and the header was never updated. Eric is the one who wrote all of this code

Reply:
Thanks Tyler.